src/modeler/AutoRest.Swagger/OperationBuilder.cs

```diff
@@ -148,9 +148,15 @@ public Method BuildMethod(HttpMethod httpMethod, string url, string methodName,
                 headerType = null;
             }
 
+            
             // Response format
             List<Stack<IModelType>> typesList = BuildResponses(method, headerType);
 
+            method.IsXNullableReturnType = (_operation.Responses.Keys.All(key =>
```

Comment:
a little over-parenthesized (outer one is superfluous), would also go with expression-only lambda notation